docs/source/reference/kubernetes/index.rst

```diff
@@ -1,30 +1,29 @@
 .. _kubernetes-overview:
 
-SkyPilot on Kubernetes (Alpha)
-==============================
+Deploying on Kubernetes (Alpha)
+======================================
 
 .. note::
-    Kubernetes support for SkyPilot is an alpha preview under active development.
+    Kubernetes support is in alpha preview and under active development.
     There may be rough edges and features may change without notice.
     Please report any `bugs <https://github.com/skypilot-org/skypilot/issues>`_ and
     `reach out to us <http://slack.skypilot.co>`_ for feature requests.
 
-SkyPilot tasks can be deployed on your private on-prem or cloud clusters running Kubernetes.
+SkyPilot tasks can be deployed on your private on-prem or cloud Kubernetes clusters.
 The Kubernetes cluster gets added to the list of "clouds" in SkyPilot and SkyPilot
 tasks can be submitted to your Kubernetes cluster just like any other cloud provider.
 
-**Benefits of bringing your Kubernetes cluster to SkyPilot:**
+**Benefits of using SkyPilot to run jobs on your Kubernetes cluster:**
 
 * Get SkyPilot features (setup management, job execution, queuing, logging, SSH access) on your Kubernetes resources
```

Comment:
```suggestion
* Get ergonomic SkyPilot features (setup management, job execution, queuing, logging, SSH access) on your Kubernetes resources
```

Reply:
Not sure about adding ergonomic - sounds a little fluffy. The convenience is covered in the examples in brackets and the next bullet point.